source/GooglePlayServicesProject.cshtml

```diff
@@ -268,15 +283,19 @@
     @if (@Model.NuGetPackageId == "Xamarin.GooglePlayServices.SafetyNet")
     {
       <Reference Include="System.Net.Http" />
-      <PackageReference Include="System.Text.Json" Version="5.0.2" />
+      <PackageReference Include="System.Text.Json" Version="6.0.0" />
     }
     @if (@Model.NuGetPackageId == "Xamarin.Firebase.Firestore")
     {
-      <PackageReference Include="Xamarin.Google.Guava" Version="28.2.0.1" />
+      <PackageReference Include="Xamarin.Google.Guava" Version="29.0.0" />
     }
     @if (@Model.NuGetPackageId == "Xamarin.Firebase.Crashlytics")
     {
-      <PackageReference Include="Xamarin.Google.Dagger" Version="2.27.0" />
+      <PackageReference Include="Xamarin.Google.Dagger" Version="2.37.0" />
+    }
+    @if (@Model.NuGetPackageId == "Xamarin.Firebase.InAppMessaging")
```

Comment:
Let's remove this whole file from the PR.  Adding unrelated changes (even commented out) to PR's causes unnecessary review time and unnecessary merge conflicts.

In this case, this new `<PackageReference>` is not commented out and causes additional diffs and questions of why we would want to add this instead of using the automatic dependencies.